index.js

```diff
@@ -1587,6 +1597,32 @@ ContainerWindow.prototype = {
 
     return true;
   },
+
+  _shutdownContainers() {
+    ContextualIdentityProxy.getIdentities().forEach(identity => {
+      if (IDENTITY_ICONS_STANDARD.indexOf(identity.icon) !== -1 &&
+          IDENTITY_COLORS_STANDARD.indexOf(identity.color) !== -1) {
+        return;
+      }
+
+      if (identity.userContextId <= IDENTITY_ICONS_STANDARD.length) {
+        identity.icon = IDENTITY_ICONS_STANDARD[identity.userContextId - 1];
+      } else {
+        identity.icon = IDENTITY_ICONS_STANDARD[0];
```

Comment:
We could actually check if the current icon doesn't exist in `IDENTITY_ICONS_STANDARD` which would keep icons how the user modified them.

Same for colors too.